src/refmt_impl.ml

```diff
@@ -209,8 +209,9 @@ let () =
           raise (Invalid_config ("Invalid --parse setting for interface '" ^ s ^ "'."))
         )
       in
-      let _ = if parsedAsInterface then
-          raise (Invalid_config ("The file parsed does not appear to be an implementation file.")) in
+      let _ =
+          if parsedAsInterface then
+              raise (Invalid_config ("The file parsed does not appear to be an implementation file.")) in
```

Comment:
Maybe move 'in' to the next line so it is more consistent? 

(This is why we need reason to automatically make the file consistent!)